packages/loot-core/package.json

```diff
@@ -24,7 +24,7 @@
     "@rschedule/json-tools": "^1.2.0",
     "@rschedule/standard-date-adapter": "^1.2.0",
     "absurd-sql": "0.0.53",
-    "better-sqlite3": "7.4.1",
+    "better-sqlite3": "^8.0.1",
```

Comment:
Necessary to do this if we upgrade `electron-rebuild`. However it then breaks one unit test. Need to investigate what's going on there.

Reply:
Fix: https://github.com/actualbudget/actual/pull/714